xcode/Subconscious/Shared/Models/Petname.swift

```diff
@@ -71,9 +71,12 @@ public struct Petname:
     
     /// Join a list of petnames into a dotted string, i.e. [foo, bar, baz] -> foo.bar.baz
     /// Names are joined in order of their appearance in `petnames`
-    public init?(petnames: [Petname]) {
-        let petnamePath = petnames.map({ s in s.verbatim }).joined(separator: ".")
-        self.init(petnamePath)
+    public init(petnames: [Petname]) {
```

Comment:
Update so initializer no longer needs to be failable. The petnames are already parsed, so no need to fail.